docs/score-sortition-flow.md

```
@@ -0,0 +1,377 @@
+```
+
+**All Ciphernodes:**
+- `CiphernodeRegistrySolReader` picks up `CommitteePublished` event
+- Broadcasts as `EnclaveEvent::CommitteePublished`
+- Committee is now officially registered and ready for computations
+
+---
+
+## Architecture Diagram
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)